model/clusters_mgmt/v1/resource_resource.model

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages currently available cluster resources 
+resource Resource {
+  // Retrieves currently available cluster resources 
+  method List {
+    // Index of the requested page, where one corresponds to the first page.
+		in out Page Integer = 1
```

Comment:
Indentation is inconsistent, make sure you use tabs for indentation, rather than spaces